### PR TITLE
Use Prettier on Templates

### PR DESCRIPTION
**Pull request by jrjohnson**
Activating `ember-template-lint-plugin-prettier` and running the fixer on all templates, including those in tests.

### CODE REVIEW

File: packages/test-app/tests/integration/components/truncate-text-test.js [hunk 1]
```diff
@@ -8,138 +8,107 @@ module('Integration | Component | truncate-text', function (hooks) {
...
 
-    await render(hbs`
-      <TruncateText>
-        {{this.shortText}}
-      </TruncateText>
-
-`);
+    await render(hbs`<TruncateText>
+  {{this.shortText}}
+</TruncateText>`);
```
Comment (michaelchadwick) [inline packages/test-app/tests/integration/components/truncate-text-test.js:29]: I guess this is an improvement, but it still seems awkward.

File: packages/test-app/tests/integration/components/truncate-text-test.js [hunk 2]
```diff
@@ -8,138 +8,107 @@ module('Integration | Component | truncate-text', function (hooks) {
...
 
   test('it renders empty', async function (assert) {
-    await render(hbs`<TruncateText />
-`);
+    await render(hbs`<TruncateText />`);
     assert.strictEqual(component.text, '');
 
-    await render(hbs`<TruncateText></TruncateText>
-`);
+    await render(hbs`<TruncateText />`);
```
[the remark this answers is not in the document]
Comment (jrjohnson) [reply packages/test-app/tests/integration/components/truncate-text-test.js:14]: I'm surprised this was done too, these are identical expressions though. I guess it makes sense?

**Comment by jrjohnson** [comment]
Lots of Percy diffs, some of them seem relevant and weird. Going to dig in further. Might just be that we need a new set of current screen shots to compare to.

**Comment by jrjohnson** [comment]
Hand checked a bunch of these, and Percy is just out of sync with all the big changes we made today. Will re-run this tomorrow and compare to the nightly update from building `master`.

File: packages/test-app/tests/integration/components/truncate-text-test.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (stopfstedt) [reply packages/test-app/tests/integration/components/truncate-text-test.js:14]: less is more. i'm good with that. go prettier!

**Review by stopfstedt — APPROVED** [review]
looks better to me.

File: packages/test-app/tests/integration/components/truncate-text-test.js (hunk 1 above)
Comment (jrjohnson) [reply packages/test-app/tests/integration/components/truncate-text-test.js:29]: One of the best/worst things about prettier is that it makes a bunch of choices for you. Sometimes they're awesome, sometimes you hate them. I try and lean into the good part - we don't have to make any of these decisions ourselves.

**Review by michaelchadwick — APPROVED** [review]
If tests pass, and the formatting of templates is consistent, I can work with it.

**Pull request merged**